main.tf

```diff
@@ -32,4 +40,5 @@ resource "datadog_monitor" "default" {
       critical_recovery = try(each.value.thresholds["critical_recovery"], null)
     }
   }
+
```

Comment:
```suggestion
```
Could you please remove this newline?

Reply:
Guess this line should be between the two dynamic blocks as the are now glued together...